plugins/inputs/internal/internal.go

```diff
@@ -2,25 +2,31 @@ package internal
 
 import (
 	"runtime"
+	"strings"
 
 	"github.com/influxdata/telegraf"
+	inter "github.com/influxdata/telegraf/internal"
 	"github.com/influxdata/telegraf/plugins/inputs"
 	"github.com/influxdata/telegraf/selfstat"
 )
 
 type Self struct {
 	CollectMemstats bool
+	SendVersion     bool
```

Comment:
Deprecate `CollectMemstats` and use an include/exclude filter https://github.com/influxdata/telegraf/blob/4e59d51844dc2da4f2771eac4a9e33a986f8751b/plugins/inputs/docker_log/docker_log.go#L81-L82

https://github.com/influxdata/telegraf/blob/4e59d51844dc2da4f2771eac4a9e33a986f8751b/plugins/inputs/docker_log/docker_log.go#L398